src/cli/init/templates.ts

```diff
@@ -17,7 +17,31 @@ import { InitOptions } from "../types";
 
 export const { dependencies, devDependencies, peerDependencies, scripts, version } = packageJSON;
 
-export function genPkgJSON(opts: InitOptions, pgkVerOverride?: string) {
+type peprPackageJSON = {
+  data: {
+    name: string;
+    version: string;
+    description: string;
+    keywords: string[];
+    engines: { node: string };
+    pepr: {
+      uuid: string;
+      onError: string;
+      webhookTimeout: number;
+      customLabels: { namespace: { "pepr.dev": string } };
+      alwaysIgnore: { namespaces: never[] };
+      includedFiles: never[];
```

Comment:
```suggestion
      includedFiles: never[];
```

Should this be a string array? This is so that the package will be built with a custom file, like a wasm file so the module author can use it in the module

Reply:
Correct, that's also consistent with the docs